app/api-fetch/all-songs.ts

```
@@ -3,42 +3,85 @@
+    // try {
         if (localStorage.getItem("songs") !== null && localStorage.getItem("songsTime") === null) {
             localStorage.removeItem("songs");
         }
-        // xoá cache nếu đã quá 1 phút
+        // xoá cache nếu đã quá 1 giờ
         if (localStorage.getItem("songs") !== null || Date.now() - parseInt(localStorage.getItem("songsTime")!) < 3600000) {
-            const data = SongListSchema.parse(JSON.parse(localStorage.getItem("songs")!));
-            return data as Song[];
+            try{
+                const data = AlternativeSongListSchema.parse(JSON.parse(localStorage.getItem("songs")!));
+                console.log(data)
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)